renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": [
+        "github>konflux-ci/mintmaker//config/renovate/renovate.json"
+    ],
+    "dockerfile": {
+        "fileMatch": ["^Dockerfile\.[a-zA-Z0-9\.-]+$"]
+    },
+    "ignorePaths": ["drift-cache"],
```

Comment:
We don't have this path..is this needed regardless?

Reply:
I left a comment about this in the commit message. I was thinking about keeping this there for when we get hermetic builds. We can just re add it.